### DISCUSSION TITLE
Falcon-StartSession

### DISCUSSION THREAD

**Discussion by zgold15** [CrowdStrike/psfalcon#89] [Q&A] [answered]
I am receiving a "write-result: 40401: Could not establish sensor comms" error. There seems to be an issue in the Private.ps1 file line 582, char 35 "$Result = Write-Result $Request" if I can get some insight into this it would be appreciated.

**Comment by bk-cs** [comment, marked as answer]
`40401: Could not establish sensor comms` is an API error that is produced when you attempt to use Real-time Response with a host that isn't responding.

You can re-try your command with a device that is known to be online, or use the `-QueueOffline $true` parameter with your `Start-FalconSession` or `Invoke-FalconRtr` command.
